browserdebuggertools/chrome/interface.py

```diff
@@ -29,6 +34,7 @@ def __init__(self, port, timeout=30, domains=None):
         is a dictionary of the arguments passed with the domain upon enabling.
         """
         self._socket_handler = SocketHandler(port, timeout, domains=domains)  # type: SocketHandler
+        self._node_map = {}
```

Comment:
Could be worth leaving a TODO to clean this up or to reset it in this task.

Reply:
I've now implemented the reset